modules/user-assigned-identity/test/main.tf

```
@@ -0,0 +1,17 @@
+provider "azurerm" {
+  features {}
+}
+
+module "azurerm_user_assigned_identity" {
```

To keep in style with the others.
```suggestion
module "user_assigned_identity" {
```